pandas/core/groupby.py

```diff
@@ -1504,7 +1504,11 @@ def aggregate(self, values, how, axis=0):
                 raise NotImplementedError
             out_shape = (self.ngroups,) + values.shape[1:]
 
-        if is_numeric_dtype(values.dtype):
+        if is_datetime:
```

Comment:
Agreed... wanted to make sure to understand if that was changing that behavior for groupby aggregations.